src/Main.java

```diff
@@ -2,5 +2,62 @@ public class Main {
     public static void main(String[] args) {
         System.out.println("привет Ванечек!");
         System.out.println("Hello world!");
+
+        var dog = 8.0;
+        System.out.println(dog);
+        var cat = 3.6;
+        System.out.println(cat);
+        var paper = 763789.0;
+        System.out.println(paper);
+
+        dog = dog + 4;
+        System.out.println(dog);
+        cat = cat + 4;
+        System.out.println(cat);
+        paper = paper + 4;
+        System.out.println(paper);
+
+        dog = dog - 3.5;
+        System.out.println(dog);
+        cat = cat - 3.5;
+        System.out.println(cat);
+        paper = paper - 3.5;
+        System.out.println(paper);
+
+        var friend = 19;
+        friend = friend + 2;
+        System.out.println(friend);
+        friend = friend / 3;
+        System.out.println(friend);
+
+        var frog = 3.5;
+        frog = 3.5 * 10;
```

Comment:
```suggestion
        frog = frog * 10;
```
нужно использовать переменную